research-software-directory/RSD-as-a-service
# 895 extra organisation fields

# Add short description to organisation

Closes #895 

Changes proposed in this pull request:
* Short description property is added to organisation table
* Short description can be edited in the settings of the organisation (see image below) 
* Additional fields added to organisation table: short_description, country and lat_long  

How to test:
* `make start` to build app and generate test data
* login as rsd_admin in order to be able to edit any organisation
* navigate to settings
* confirm Short description input is present (after website link)
* add/edit short description and confirm that information is saved (on lost focus)

## Organisation settings (with short description input)

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/34620972-d8c9-43a3-ada2-2d168d939128)


PR Checklist:
